flake.nix

```diff
@@ -2,8 +2,12 @@
   description = "Hyperconverged Infratructure for NixOS";
 
   inputs = {
-    microvm.url = "github:astro/microvm.nix";
-    microvm.inputs.nixpkgs.follows = "nixpkgs";
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable"; #Why was nixpkgs missing?
```

Comment:
It't taken from the local system's nix registry but I guess pinning it here is better.